routes/paymentRouter.js

```diff
@@ -1,80 +1,39 @@
 // Set your secret key: remember to change this to your live secret key in production
 // See your keys here: https://dashboard.stripe.com/account/apikeys
-const router = require('express').Router()
-const validatePaymentInput = require('../middleware/validatePaymentData')
-const validatePaymentCompleteInput = require('../middleware/validatePaymentComleteData')
-const Store = require('../models/store')
-const Cart = require('../models/cart')
+const router = require('express').Router();
+const stripeConfig = require('../config.stripe');
+const uuid = require('uuid');
 
-const stripe = require('stripe')(process.env.STRIPE_SECRET)
+const stripe = require('stripe')(stripeConfig.stripe.secretKey);
+router.post('/create-payment-intent', async (req, res) => {
+	try {
+		const { price, pK } = req.body;
 
-router.post('/charge', async (req, res) => {
-  const { errors, isValid } = validatePaymentInput(req.body)
-  if (!isValid) {
-    return res.status(400).json(errors)
-  }
-  try {
-    const transaction = req.body
-    const store = await Store.findById({ _id: transaction.storeId })
-    const stripeId = store.stripeId
-    let currency
+		const idempontencyKey = uuid();
 
-    switch (store.currency) {
-      case 'DOL':
-        currency = 'usd'
-        break
-      case 'POU':
-        currency = 'gbp'
-        break
-      case 'EUR':
-        currency = 'eur'
-        break
-      case 'YEN':
-        currency = 'jpy'
-        break
-      default:
-        currency = 'usd'
-    }
+		const paymentIntentActual = await stripe.paymentIntents.create(
+			{
+				amount: price,
+				currency: 'usd'
+			}
+			// {
+			// 	idempontencyKey: idempontencyKey
+			// }
+		);
 
-    const paymentIntent = await stripe.paymentIntents.create({
-      amount: transaction.amount * 100,
-      currency: currency
-    }
-    , {
-      stripeAccount: stripeId
-    }
-    )
-    res.status(200).json({ paymentIntent: paymentIntent, stripeId })
-  } catch (error) {
-    res.status(400).json(error)
-  }
-})
+		console.log('paymentIntent', paymentIntentActual);
```

Comment:
Remove this console log if it's not needed

Reply:
this should be removed now.